src/main/java/use_case/TaskGetter.java

```diff
@@ -42,4 +42,14 @@
 
         return task_data;
     }
+
+    public Task getTaskByName(String name, AccessTodoData accessTodoData) {
+        List<Task> allTasks = accessTodoData.getTodoList().getTasks();
+        for (Task task : allTasks) {
+            if (task.getTaskName().equals(name)) {
+                return task;
```

Comment:
Seems that `name` is used as id everywhere. So for now, when the user wants to create a new task or event, he will not allowed to create a repeated name. This probably requires less work to fix.

Reply:
OK this does not need to be fixed for phase 0 so we can choose a better method to fix it later

Reply:
this method is temporary. After we have implemented the snowflake id thing we could binary-search the tasks